usb/backend/libusb0.py

```diff
@@ -487,10 +493,6 @@ def set_configuration(self, dev_handle, config_value):
         _check(_lib.usb_set_configuration(dev_handle, config_value))
 
     @methodtrace(_logger)
-    def set_interface_altsetting(self, dev_handle, intf, altsetting):
-        _check(_lib.usb_set_altinterface(dev_handle, altsetting))
-
-    @methodtrace(_logger)
     def get_configuration(self, dev_handle):
```

Comment:
I moved it so that the `libusb0` implements `IBackend` in the same order as the `IBackend` interface is defined. Not strictly needed, just easier for readability when comparing `libusb0` vs `libusb1` vs `IBackend`.